themes/default/content/docs/guides/saml/onelogin.md

```diff
@@ -0,0 +1,113 @@
+---
+title: OneLogin
+meta_desc: This page provides a walkthrough important aspects of configuring
+  OneLogin as a SAML SSO identity provider (IdP).
+menu:
+  userguides:
+    parent: saml
+    weight: 2
+
+aliases:
+  - /docs/reference/service/saml-onelogin/
+  - /docs/console/accounts/saml/onelogin/
+---
+
+This guide walks you through configuring OneLogin as a SAML SSO identity provider (IdP) for the Pulumi Console.
+
+## Prerequisites
+
+- [Single Sign-On]({{< relref "sso" >}})
+
+## Creating the OneLogin Application
+
+The first step is to create a new OneLogin Application for Pulumi SSO:
+
+- From the OneLogin Administration portal, go to the **Applications** page and click the **Add App** button.
+- Search for `SAML Test Connector (Advanced)` and select it.
+
+![Finding the SAML Test Connector App](/images/docs/reference/service/saml-onelogin/onelogin-find-app.png)
+
+- Enter a _Display Name_ and optionally a logo. See [Pulumi Logos](https://www.pulumi.com/brand/#logos).
+- Click **Save**
+
+![Creating a OneLogin Application example](/images/docs/reference/service/saml-onelogin/onelogin-create-saml-app.png)
+
+### Configuring the OneLogin Application
+
+Now configure the OneLogin Application with the SAML settings for Pulumi SSO.
+
+#### Configure SAML URLs
+
+Select the **Configuration** view for the application and enter/confirm the values in the following table.
+
+{{< saml-warning >}}
+
+| Configuration Settings     | Value                                                     |
+| -------------------------- | --------------------------------------------------------- |
+| Relay State                | `https://api.pulumi.com/login/acmecorp/sso`               |
+| Audience (EntityID)        | `https://api.pulumi.com/login/acmecorp/sso/saml/metadata` |
+| Recipient                  | `https://api.pulumi.com/login/acmecorp/sso/saml/acs`      |
+| ACS Consumer URL Validator | `.*`                                                      |
+| ACS Consumer URL           | `https://api.pulumi.com/login/acmecorp/sso/saml/acs`      |
+| SAML initiator             | `OneLogin`                                                |
+| SAML nameID format         | `Email`                                                   |
+| SAML issuer type           | `Specific`                                                |
+| SAML signature element     | `Response`                                                |
+| SAML encryption method     | `TRIPLEDES-CBC`                                           |
+
+> **Important:** Do not change the value of the Name ID Format once your users have started using Pulumi---not even switching its value between `EmailAddress` or `Persistent`.
```

Comment:
```suggestion
{{% notes type="warning" %}}
Do not change the value of the Name ID Format once your users have started using Pulumi---not even switching its value between `EmailAddress` or `Persistent`.
{{% /notes %}}
```

Can you add a bit more "why" to the end of this warning (e.g. what will happen if they change the value?)

Reply:
TBH, I don't know - this is text I copied from other SSO SAML docs pages (e.g. Okta).

Reply:
Gotcha. We can leave it as is for now.